models/demos/t3000/llama2_70b/scripts/llama_test_suite_T3000.sh

```diff
@@ -6,18 +6,13 @@ set -e
 # change directory to TT_METAL_HOME
 cd $TT_METAL_HOME
 
-# Run 8-chip tests on T3000s
-pytest models/demos/t3000/llama2_70b/tests/test_llama_mlp.py::test_LlamaMLP_inference[decode-8chip-T3000]
-pytest models/demos/t3000/llama2_70b/tests/test_llama_attention.py::test_LlamaAttention_inference[decode-8chip-T3000]
-pytest models/demos/t3000/llama2_70b/tests/test_llama_decoder.py::test_LlamaDecoder_inference[decode-8chip-T3000]
-pytest models/demos/t3000/llama2_70b/tests/test_llama_model.py::test_LlamaModel_inference[decode-8chip-T3000-1L]
-
-pytest models/demos/t3000/llama2_70b/tests/test_llama_mlp.py::test_LlamaMLP_inference[prefill_128-8chip-T3000]
-pytest models/demos/t3000/llama2_70b/tests/test_llama_attention.py::test_LlamaAttention_inference[prefill_128-8chip-T3000]
-pytest models/demos/t3000/llama2_70b/tests/test_llama_decoder.py::test_LlamaDecoder_inference[prefill_128-8chip-T3000]
-pytest models/demos/t3000/llama2_70b/tests/test_llama_model.py::test_LlamaModel_inference[prefill_128-8chip-T3000-1L]
+# Set Llama2_70b related cached files and tests
+export LLAMA_CKPT_DIR="/home/llama-data-repacked-2/llama-2-70b/"
+export LLAMA_TOKENIZER_PATH="/home/llama-data/tokenizer.model"
+export LLAMA_CACHE_PATH="/home/llama-data-cache/weights-cache-4"
 
-pytest models/demos/t3000/llama2_70b/tests/test_llama_mlp.py::test_LlamaMLP_inference[prefill_2k-8chip-T3000]
-pytest models/demos/t3000/llama2_70b/tests/test_llama_attention.py::test_LlamaAttention_inference[prefill_2k-8chip-T3000]
-pytest models/demos/t3000/llama2_70b/tests/test_llama_decoder.py::test_LlamaDecoder_inference[prefill_2k-8chip-T3000]
-pytest models/demos/t3000/llama2_70b/tests/test_llama_model.py::test_LlamaModel_inference[prefill_2k-8chip-T3000-1L]
+# Run 8-chip tests on T3000s
+pytest models/demos/t3000/llama2_70b/tests -k test_llama_mlp_t3000
```

Comment:
Could you pls rewrite these targets to `.py` files, and absolutely no `-k`? We're going through a bit of a refactor with command rules. We'll be announcing these once they're final

Same for `run_frequent_regressions_multi_device.sh`